### ISSUE TITLE
Trojan.GenericKD.48366396

### ISSUE THREAD

**Issue by whiteGauna** [Rangi42/polished-map#72]
when I want to download the polished map exe from 4.71 I get the warning from my Security software, that the files inclused the file Trojan.GenericKD.48366396.

**Comment by mk-pmb** [comment]
The "Trojan.GenericKD.48366396" match seems to be common for BitDefender in a lot of files. Does it give any more information what exactly it deems dubious about that file?
It might be a classic case of some innocent library getting marked as dubious just because some random virus happens to use the same library. If in doubt, better download the source code, check it for evil stuff, and compile it yourself.

**Comment by Rangi42** [comment]
Yeah, false positives from antivirus software are pretty common -- applying machine learning to uncommon or brand-new .exes does that.
